docs: refine development document (#2)
chaos-mesh/website: docs/development_guides/dev_hello_world.md

Now take the following steps to run chaos:

1. Create namespace `chaos-testing`

    ```bash
    kubectl create namespace chaos-testing
    ```

2. Get the related custom resource type for Chaos Mesh:

    ```bash
    kubectl apply -f manifests/
    ```

    You can see CRD `helloworldchaos` is created:

    ```log
    customresourcedefinition.apiextensions.k8s.io/helloworldchaos.chaos-mesh.org created
    ```

>     And then you can get it as follows:

Reviewer: ```suggestion
    Now you can get the CRD using the command below:
```

Author: done

    ```bash
    kubectl get crd helloworldchaos.chaos-mesh.org
    ```

3. Install Chaos Mesh:

    - For helm 3.X

      ```bash
      helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
      ```

    - For helm 2.X

      ```bash
      helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
      ```

>     Then you get pods from namespace `chaos-testing` to make sure chaos-mesh is installed:

Reviewer: ```suggestion
    To verify your installation, get pods from the `chaos-testing` namespace:
```

Author: done

    ```bash
    kubectl get pods --namespace chaos-testing -l app.kubernetes.io/instance=chaos-mesh

>     ```

Reviewer: Is this part of installing chaos mesh?

Author: yes, just make sure it installs success

    > **Note:**
    >

>     > The arguments `--set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock` is used to to support network chaos on kind.

Reviewer: ```suggestion
    > Arguments `--set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock` are used to to support network chaos on kind.
```

Author: done

4. Create `chaos.yaml` in any location with the lines below:

    ```yaml
    apiVersion: chaos-mesh.org/v1alpha1
    kind: HelloWorldChaos
    metadata:
      name: hello-world
      namespace: chaos-testing
    spec: {}
    ```

5. Apply the chaos:

    ```bash
    kubectl apply -f /path/to/chaos.yaml
    ```

    ```bash
    kubectl get HelloWorldChaos -n chaos-testing
    ```

    Now you should be able to check the `Hello World!` result in the log:

    Now you can check the log of `chaos-controller-manager`:

    ```bash
    kubectl logs chaos-controller-manager-{pod-post-fix} -n chaos-testing
    ```

    The log is as follows:

    ```log
    2020-09-07T09:21:29.301Z        INFO    controllers.HelloWorldChaos     Hello World!    {"reconciler": "helloworldchaos"}
    2020-09-07T09:21:29.308Z        DEBUG   controller-runtime.controller   Successfully Reconciled {"controller": "helloworldchaos", "request": "chaos-testing/hello-world"}
    ```

    > **Note:**
    >

>     > `{pod-post-fix}` is a random string generated by Kubernetes, you can check it by executing `kubectl get pod -n chaos-testing`.

Reviewer: ```suggestion
    > `{pod-post-fix}` is a random string generated by Kubernetes. You can check it by executing `kubectl get pod -n chaos-testing`.
```

Author: done
